ui/dashboards/src/components/UnsavedChangesConfirmationDialog/UnsavedChangesConfirmationDialog.tsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```

Comment:
Is this eslint-disable line still needed?

Reply:
oh good catch thanks